toolkit/tools/internal/network/network.go

```diff
@@ -4,166 +4,15 @@
 package network
 
 import (
-	"context"
```

Comment:
It might be worth running `go mod tidy` to see if any dependencies are now gone.

Reply:
no changes